src/js/tech/tech.js

```diff
@@ -551,7 +551,7 @@ class Tech extends Component {
       // passed in
       const script = document.createElement('script');
 
-      script.src = this.options_['vtt.js'] || 'https://cdn.rawgit.com/gkatsev/vtt.js/vjs-v0.12.1/dist/vtt.min.js';
+      script.src = this.options_['vtt.js'] || 'https://cdn.rawgit.com/gkatsev/vtt.js/vjs-v0.12.3/dist/vtt.min.js';
```

Comment:
Looking into getting it published to the CDN.
Also, I realized that we probably want to update vttjs to get rid of `browser.js` and only use `browser-index.js`.